Extensions/xkit_preferences.js

```diff
@@ -950,7 +950,9 @@ XKit.extensions.xkit_preferences = new Object({
 
 			for(var extension in mdata.extensions) {
 
-				m_html = m_html + XKit.extensions.xkit_preferences.gallery_parse_item(mdata.extensions[extension]);
+				if(mdata.extensions[extension].hidden != "true"){
```

Comment:
There is absolutely no reason to introduce a new attribute to hide an extension in the gallery.
We have a blacklist you just add the extension to, see https://github.com/new-xkit/XKit/blob/master/Extensions/xkit_preferences.js#L1093